### PR TITLE
boards: Improve documentation for pic32-clicker and pic32-wifire

### PR DESCRIPTION
**Pull request by francois-berder**
### Contribution description

creatordev.io is no more so all links in pic32-clicker and pic32-wifire documentation files are dead. I took the opportunity to update the documentation for these two boards.

### Testing procedure

None

### Issues/PRs references

None

### CODE REVIEW

File: boards/pic32-clicker/doc.txt [hunk 1]
```diff
@@ -3,16 +3,91 @@
...
-https://docs.creatordev.io/clicker/guides/quick-start-guide/#programming-a-6lowpan-clicker
-
-The RIOT build generates a hexfile compatible with MPLAB-IPE.
+The `6lowpan clicker` is an evaluation board by Mikroe featuring a
+PIC32MX470F512H, a 6lowpan radio (CA810) by Cascoda and a Mikrobus
+socket for expansion boards.
 
 More general information on the board and related documentation can be found
-here:
+here: https://www.mikroe.com/clicker-6lowpan. Schematics for the board can be
```
Comment (smlng) [inline boards/pic32-clicker/doc.txt]: for readability and *nicer* HTML output: please use proper Markdown links, i.e. [here](<url>) or [manual](<url>) below.

File: boards/pic32-wifire/doc.txt [hunk 2]
```diff
@@ -3,19 +3,88 @@
 @ingroup     boards
 @brief       Support for the Digilent PIC32 WiFire
 
-For instructions on reflashing see:
+## Overview
 
-https://docs.creatordev.io/wifire/guides/wifire-programming/
+The ChipKit Wifire is a development board by Digilent featuring a PIC32MZ and a Wifi module (MRF24WG0MA).
+General information about the board can be found here: https://store.digilentinc.com/wi-fire-wifi-enabled-pic32mz-microcontroller-board/
```
Comment (smlng) [inline boards/pic32-wifire/doc.txt]: use Markdown links (see above)

File: boards/pic32-wifire/doc.txt [hunk 3]
```diff
@@ -3,19 +3,88 @@
...
+Please do not use board rev A or B as they use a different MCU (PIC32MZ2048ECG100).
 
-The RIOT build generates a hexfile compatible with MPLAB-IPE.
+Schematics for the board (rev C): https://reference.digilentinc.com/_media/reference/microprocessor/wi-fire/chipkit_wifire_sch.pdf
```
Comment (smlng) [inline boards/pic32-wifire/doc.txt]: use bullet list for both schematics and also Markdown links [rev C](<url>) ... [rev D](<url>)

File: boards/pic32-clicker/doc.txt (hunk 1 above)
Comment (francois-berder) [reply boards/pic32-clicker/doc.txt]: Fixed

File: boards/pic32-wifire/doc.txt (hunk 2 above)
Comment (francois-berder) [reply boards/pic32-wifire/doc.txt]: Fixed

File: boards/pic32-wifire/doc.txt (hunk 3 above)
Comment (francois-berder) [reply boards/pic32-wifire/doc.txt]: Fixed

**Comment by francois-berder** [comment]
> please squash

Done

**Pull request merged**